
sub_odometry_topic_         : /turtlebot/kobuki/odom_ground_truth
sub_path_topic_             : /path
sub_joystick_topic_         : /joy
sub_speed_topic_            : /speed
sub_stop_topic_             : /stop

pub_speed_topic_            : /cmd_vel

kSensorOffsetX    : 0.0
kSensorOffsetY    : 0.0
kSubSkipNum       : 1
kTwoWayDrive      : true
kLookAheadDis     : 0.5
# yawRateGain     : 7.5
# stopYawRateGain : 7.5
kYawRateGain      : 1.5
kStopYawRateGain  : 7.5
# maxYawRate      : 90.0
kMaxYawRate       : 20.0
kMaxSpeed         : 0.2
# maxAccel        : 2.5
kMaxAccel         : 1.5
kSwitchTimeThre   : 1.0
kDirDiffThre      : 0.5
kStopDisThre      : 0.2
kSlowDwnDisThre   : 0.85
kUseInclRateToSlow: false
kInclRateThre     : 120.0
kSlowRate1        : 0.25
kSlowRate2        : 0.5
kSlowTime1        : 2.0
kSlowTime2        : 2.0
kUseInclToStop    : false
kInclThre         : 45.0
kStopTime         : 5.0
kNoRotAtStop      : false
kNoRotAtGoal      : true
kAutonomyMode     : true
kAutonomySpeed    : 2.0
kJoyToSpeedDelay  : 2.0